src/index.html

```diff
@@ -1,19 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width,
-          user-scalable=no,
-          initial-scale=1.0,
-          maximum-scale=1.0,
-          minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width,
+            user-scalable=no,
+            initial-scale=1.0,
+            maximum-scale=1.0,
+            minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>layout_snake</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="our-body">
 
-<script type="text/javascript" src="./main.js"></script>
-</body>
+    <section class="block block1">
+      <div class="number number1">
```

Comment:
no need to place such a short text on the new line

Reply:
Don't understand

Reply:
sorry, don't understand